src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -88,6 +88,12 @@ public string ClientVersion
 
         public bool LegacyCacheCompatibilityEnabled { get; internal set; } = true;
 
+        /// <summary>
+        /// Currently this is only required for WAM
+        /// </summary>
+        public bool IsMsaPassthrough { get; set; } = false;
+
```

Comment:
what happens if a 3rd party tries to use this or it's used w/out WAM? just wondering.

Reply:
should it be tied to the experimental flag as well? like WAM?

Reply:
I made it experimental.

Reply:
if 3rd party uses it .. hmm... let's see... A bit obscure exception but makes sense to me. It essentially states "I could not find an AAD tenant for this MSA user" ... which is normal for a non-MSA-PT configuration. 

WAM_unexpected_aad_error. Possible causes: no Internet connection or invalid redirect uri - please see https://aka.ms/msal-net-wam Details: V2Error: invalid_request AADSTS90019: No tenant-identifying information found in either the request or implied by any provided credentials.